lib/Runtime/Library/JavascriptError.cpp

```diff
@@ -484,6 +582,12 @@ namespace Js
         pError->m_errorType = errorType;
     }
 
+    void JavascriptAggregateError::SetErrorsProperties(JavascriptAggregateError* pError, RecyclableObject* errors)
```

Comment:
Could these steps be inlined when used rather than being a new method, considering how short it is?